mianzain/GITHUB_ZAIN/day4/events_lab

```diff
@@ -0,0 +1 @@
+Subproject commit 280a5670b201b8617229238c189a07b3b66f1280
```

Comment:
@mianzain 
This 'Subproject commit' often indicates that you copied another github repository under your 23-modules repository. Unfortunately that won't work. If this is a lab that you were asked to fork, please remove it from here and put it in your `dev/` directory. Then push to the fork and make a pull-request on **that lab's repo**

If this is just a few files you can instead remove the hidden .git directory from `23-homework/jsnow/some_lab/.git` (warning don't remove `23-modules/.git`)  Then commit on this repo and re-push.  If that doesn’t make the files available to add in git you’ll also have to `git rm —cached some_lab`  Make sure you back up the files first.